crates/nu-plugin-engine/src/persistent.rs

```diff
@@ -52,6 +52,8 @@ struct RunningPlugin {
     interface: PluginInterface,
     /// Garbage collector for the plugin
     gc: PluginGc,
+    /// RAII guard for this plugin's ctrl-c handler
+    _ctrlc_guard: Option<ctrlc::Guard>,
```

Comment:
Hmm, is there a good reason to start the field name with an underscore? I guess that might mean when destructuring, you keep the guard in scope without getting a warning, which would be nice... but it just seems unusual.

Reply:
The guard just needs to be held, but it's never used, so without the underscore we get:

```
warning: field `ctrlc_guard` is never read
  --> crates/nu-plugin-engine/src/persistent.rs:56:5
  ```

Reply:
I suppose that's okay. You could also just do that for the variable binding (e.g. `ctrlc_guard: guard`), or explicitly `drop(ctrlc_guard)` at the end